### PR TITLE
Fix reference to Microsoft.NetCore.App

### PR DESCRIPTION
**Pull request by pranavkm**
cc @moozzyk \ @pakrym

### CODE REVIEW

File: src/Microsoft.AspNetCore.Server.IISIntegration.Tools/project.json [hunk 1]
```diff
@@ -14,17 +14,15 @@
     "Microsoft.DotNet.ProjectModel": "1.0.0-*",
     "System.Diagnostics.Process": "4.1.0-*"
   },
-
   "frameworks": {
     "netcoreapp1.0": {
       "imports": [
-        "portable-net451+win8",
-        "dnxcore50"
```
Comment (pranavkm) [inline src/Microsoft.AspNetCore.Server.IISIntegration.Tools/project.json]: This is no longer necessary.

File: src/Microsoft.AspNetCore.Server.IISIntegration.Tools/project.json [hunk 2]
```diff
@@ -14,17 +14,15 @@
     "Microsoft.DotNet.ProjectModel": "1.0.0-*",
     "System.Diagnostics.Process": "4.1.0-*"
   },
-
   "frameworks": {
     "netcoreapp1.0": {
       "imports": [
-        "portable-net451+win8",
```
Comment (pranavkm) [inline src/Microsoft.AspNetCore.Server.IISIntegration.Tools/project.json]: Switching to full form to remove the odds of accidentally importing incorrect dependencies

File: src/Microsoft.AspNetCore.Server.IISIntegration.Tools/project.json [hunk 3]
```diff
@@ -14,17 +14,15 @@
...
-        "dnxcore50"
+        "portable-dnxcore50+net45+win8+wp8+wpa81"
       ],
       "dependencies": {
         "Microsoft.NETCore.App": {
           "type": "platform",
-          "version": "1.0.0-rc2-23931"
+          "version": "1.0.0-*"
```
Comment (pranavkm) [inline src/Microsoft.AspNetCore.Server.IISIntegration.Tools/project.json]: Does doing * break some scenario? It was changed in https://github.com/aspnet/IISIntegration/commit/362c093174d36686ded81ab7e76c3f9a729030b2

Comment (pakrym) [inline src/Microsoft.AspNetCore.Server.IISIntegration.Tools/project.json]: I think it should be * here

**Comment by moozzyk** [comment]
Looks good

File: src/Microsoft.AspNetCore.Server.IISIntegration.Tools/project.json (hunk 3 above)
Comment (GuardRex) [inline src/Microsoft.AspNetCore.Server.IISIntegration.Tools/project.json]: Breaks here. 
```
Error loading hostpolicy 1.0.1-rc2-002330; interface mismatch between hostpolicy [32771] and hostfxr [120]
Specifically, the structure of corehost_init_t has changed, do not know how to interpret it
```

Comment (pranavkm) [inline src/Microsoft.AspNetCore.Server.IISIntegration.Tools/project.json]: @GuardRex - I don't see this issue locally (and nor does AppVeyor). Are you sure you have the right CLI in the path (right = whatever's in https://github.com/aspnet/KoreBuild/blob/release/build/cli.version.win)

Comment (GuardRex) [inline src/Microsoft.AspNetCore.Server.IISIntegration.Tools/project.json]:
```
.NET Command Line Tools (1.0.0-rc2-002392)

Product Information:
 Version:     1.0.0-rc2-002392
 Commit Sha:  880dcf2fc8

Runtime Environment:
 OS Name:     Windows
 OS Version:  10.0.10586
 OS Platform: Windows
 RID:         win10-x64
```
I'll try again this morning and see if I can get it to work.

**Pull request closed without merging**